zigzag/link_generation_facade.py

```diff
@@ -24,9 +24,9 @@ def __init__(self, mediator):
                 split = urlsplit(self._get_testsuite_prop('REPO_URL'))
                 path = self._strip_git_ending(split.path)
                 self._git_sha = self._get_testsuite_prop('MOLECULE_GIT_COMMIT')
-                self._molecule_name = self._get_testsuite_prop('MOLECULE_TEST_REPO')
                 self._molecule_scenario = self._get_testsuite_prop('MOLECULE_SCENARIO_NAME')
-                self._repo_fork, self._repo_name = list(filter(None, path.split('/')))
+                self._repo_fork = list(filter(None, path.split('/')))[0]
+                self._repo_name = self._get_testsuite_prop('MOLECULE_TEST_REPO')
```

Comment:
The name here changed to align with how we build links for mk8s.  In the workflow for ASC the `_repo_name` is the  molecule repo.